test/cli/lsp-without-root-path/lsp-without-root-path.out

```diff
@@ -0,0 +1,5 @@
+Content-Length: 350
+
+{"jsonrpc":"2.0","id":0,"requestMethod":"initialize","result":{"capabilities":{"textDocumentSync":1,"hoverProvider":true,"completionProvider":{"triggerCharacters":["."]},"definitionProvider":true,"typeDefinitionProvider":true,"referencesProvider":true,"documentHighlightProvider":false,"documentSymbolProvider":false,"workspaceSymbolProvider":true}}}Content-Length: 65

```

Comment:
I try to avoid tests of this shape as they are hard to debug and are fragile to small changes in output (e.g., parameter re-ordering).

Could you try copying what we do in the `MonacoInitialization` test?

https://github.com/sorbet/sorbet/blob/49842cb1/test/lsp/protocol_test_corpus.cc#L300` 

That test also makes sure that typechecking errors are reported properly.

You can run it locally with:

```
bazel test //test/lsp:protocol_test_corpus --config=dbg
```

Once compiled, you can also run a specific test with the following, since that test corpus takes awhile to run:

```
./bazel-bin/test/lsp/protocol_test_corpus --gtest_filter="*MonacoInitialization*"
```